### DISCUSSION TITLE
Failed to package DLL into exe

### DISCUSSION THREAD

**Discussion by fattybears** [canton7/Stylet#347] [Q&A]
When I package the project into a single exe, after embedding the DLL into the EXE, the EXE cannot run. What is the possible reason?
Target framework is .NET Framework 4.7.2 and I don't set up startup object like Program.cs, maybe Bootstrapper.cs works.

**Comment by canton7** [comment]
How are you packaging it? What's the error message?

**Reply by fattybears** [reply]
I create WPF project and install Stylet.Start by Nuget. then coding..
1.Set the config file generation operation property to "embedded resources".
2.In the generated attribute, click the advanced button in the output bar, and the debug information in the dialog box is changed to "embedded".
3.Nuget install costura Fody package, set the DLL file as an embedded resource.
that's all.

**Reply by fattybears** [reply]
there is no error message

**Reply by canton7** [reply]
Check the Event Viewer, under Application. .Net crashes get recorded in there

**Reply by fattybears** [reply]
For unknown reasons, my Event Viewer doesn't work, I will reply after fixing Event Viewer, thank you anyway
